tests/python/rest_api/test_quality_control.py

```diff
@@ -1097,7 +1098,7 @@ def test_report_summary(self, task_id, tasks, jobs, quality_reports):
         assert summary["frame_share"] == summary["frame_count"] / task["size"]
 
     def test_unmodified_task_produces_the_same_metrics(self, admin_user, quality_reports):
-        old_report = max((r for r in quality_reports if r["task_id"]), key=lambda r: r["id"])
+        old_report = next(r for r in quality_reports if r["task_id"] == self.demo_task_id)
```

Comment:
Why was it changed to `next()`? It's only supposed to work with the latest task report, not any report.

Reply:
In this test class, most tests were using the `task_id` 22, so thought this should also use it, so changed it accordingly. Shouldn't all tests pass this since the unmodified tasks should produce the same metrics?

Reply:
After reverting to the original state to test it on the latest task, it's still failing.

Reply:
I was only saying about the `max` function, not the `==` part. We need to use the latest task report, not any task report.

Reply:
Ahh! Here the value being compared by max funding is report id not task Id. Due to this mistake I thought there will be a single report only so no need for max.